src/wrapped_radix_engine_toolkit/high_level/sargon_specific_types/assets_transfers/per_asset/per_asset_non_fungible_transfer.rs

```diff
@@ -0,0 +1,136 @@
+use crate::prelude::*;
+
+impl PerAssetNonFungibleTransfer {
```

Comment:
The `PerAssetNonFungibleTransfer ` type is declared with macro in `transfer_types.rs`